pkgs/applications/networking/instant-messengers/session-desktop-appimage/default.nix

```
@@ -0,0 +1,23 @@
+{ lib
+, fetchurl
+, appimageTools
+}:
+
+let
+  version = "1.6.11";
+in
+appimageTools.wrapType2 {
+  name = "session-desktop-appimage";
+  src = fetchurl {
+    url = "https://github.com/oxen-io/session-desktop/releases/download/v${version}/session-desktop-linux-x86_64-${version}.AppImage";
+    sha256 = "08vnkxj2w2k7lhygs5qhbyikpvw5nmby3qhxigbbkj5qj1wvs48b";
+  };
+
+  meta = with lib; {
+    description = "Onion routing based messenger";
+    homepage = "https://getsession.org/";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ alexnortung ];
+    platforms = platforms.linux;
```

doesn't seem to work on aarch64-linux

```suggestion
    platforms = [ "x86_64-linux" ];
```